test-packages/integration-tests/test/v2/request-builder.spec.ts

```diff
@@ -531,7 +531,7 @@ describe('Request Builder', () => {
       URL: url,
       authTokens: [],
       Authentication: 'OAuth2ClientCredentials',
-      tokenServiceURL: 'https://token.example.com',
+      tokenServiceURL: 'https://token.example.com/some/token/endpoint',
```

Comment:
For the case that the token url of the destination is used, we only have the test in the integration tests. Could we maybe add one in the unit test?

Reply:
I added a test in the xsuaa spec file